.github/workflows/pull-request-teardown.yml

```diff
@@ -17,7 +26,7 @@ jobs:
         kubectl: 1.16.2
         helm: 2.16.1
         command: |
-          export PR_NUMBER=$(echo $GITHUB_REF | awk 'BEGIN { FS = "/" } ; { print $3 }')
+          export PR_NUMBER=${{ github.event.number }}
```

Comment:
shouldn't you use the same in the deploy?

Reply:
Yes, will do this and clean up a bit if this one works.

Reply:
Added by @pameladelgado